automation_tools/satellite6/capsule.py

```diff
@@ -208,14 +208,19 @@ def generate_capsule_certs(capsule_hostname):
 
     :param str capsule_hostname: The fully qualified domain name for the
         capsule.
+    :param bool force: To force create the capsule cert if already created.
+    By default wont force create if already created.
 
     """
     cert_path = '{}-certs.tar'.format(capsule_hostname)
     result = run('[ -f {} ]'.format(cert_path), quiet=True)
-    if result.failed:
+    if result.failed or force:
         run('capsule-certs-generate -v --capsule-fqdn {} '
             '--certs-tar {}'.format(capsule_hostname, cert_path))
-    return cert_path
+    if int(run('[ -f {} ]; echo $?'.format(cert_path))) == 0:
+        return cert_path
+    else:
+        print('Warning ! Failed to generate Capsule Cert on Satellite !')
```

Comment:
We are not raising the exceptions in automation_tools. Instead we are just printing the impacts of failure and proceeding fwd.